recipes/libcurl/all/conanfile.py

```diff
@@ -113,7 +113,8 @@ def system_requirements(self):
 
     def build_requirements(self):
         if self._is_mingw and "CONAN_BASH_PATH" not in os.environ and \
-           tools.os_info.detect_windows_subsystem() != "msys2":
+           tools.os_info.detect_windows_subsystem() != "msys2" and \
+           self.settings.os_build == "Windows":
```

Comment:
Would this be fine as well?
```suggestion
        if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH") and \
                tools.os_info.detect_windows_subsystem() != "msys2":
```

Reply:
Does tools.os_info.is_windows() check the target os or the host os?

Reply:
Neither of those, the build os.

Reply:
So if they are the same so it shouldn't make a difference

Reply:
They *should* be, but `os_build` can be overridden.

See https://docs.conan.io/en/latest/systems_cross_building/cross_building.html#host-settings-os-build-arch-build-os-target-and-arch-target
conan is getting more and more crossbuild support, and because for a cross build the build system should not matter, `os_build` is getting phased out.